java/client/build.gradle

```
@@ -143,17 +144,51 @@ clean.dependsOn('cleanProtobuf', 'cleanRust')
 test.dependsOn('buildRust')
 testFfi.dependsOn('buildRust')
 
+tasks.register('copyNativeLib', Copy) {
+    from "${projectDir}/../target/release"
+    include "*.dylib", "*.so"
+    into sourceSets.main.output.resourcesDir
+}
+
+jar.dependsOn('copyNativeLib')
+copyNativeLib.dependsOn('buildRustRelease')
+compileTestJava.dependsOn('publishToMavenLocal')
```

why this dependency?

---

Without it, the gradle build fails:
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':client:compileTestJava' (type 'JavaCompile').
  - Gradle detected a problem with the following location: '/Users/jonathanlouie/Documents/ElastiCache/bq-fork/babushka/java/client/build/resources/main'.
    
    Reason: Task ':client:compileTestJava' uses this output of task ':client:copyNativeLib' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':client:copyNativeLib' as an input of ':client:compileTestJava'.
      2. Declare an explicit dependency on ':client:copyNativeLib' from ':client:compileTestJava' using Task#dependsOn.
      3. Declare an explicit dependency on ':client:copyNativeLib' from ':client:compileTestJava' using Task#mustRunAfter.
 ```

---

hmm, that is strange
At least you can replace it with 
```suggestion
compileTestJava.dependsOn('copyNativeLib')
```
as gradle suggests in p.1